dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/smallrye/metrics/adapters/SRMetricRegistryAdapter.java

```diff
@@ -62,7 +88,8 @@ public void addNameToApplicationMap(MetricID metricID) {
     public void unRegisterApplicationMetrics(String name) {
         try {
             unRegisterApplicationMetricsMethod_StringParam.invoke(srMetricRegistryObject, name);
-        } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
+        } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException
```

Comment:
Generally, it is recommended to do a null check rather than catching a NPE.

Reply:
See:
https://github.com/OpenLiberty/open-liberty/pull/22772/commits/e0dc9e57a8b31ad85ee222b2467335b7e23a96b2